concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1264,7 +1264,7 @@ Hale-1973-1798-1262	Human_Mind_and_Conditions-150	flatter	v	10b1.77	2709	FLATTER
 Hale-1973-1798-1263	Human_Mind_and_Conditions-151	flirt	v	10b1.78	494	FLIRT
 Hale-1973-1798-1264	Human_Mind_and_Conditions-152	feel at ease	v	10b1.79		
 Hale-1973-1798-1265	Human_Mind_and_Conditions-153	quarrel with words	v	10b1.80	3302	QUARREL (VERB)
-Hale-1973-1798-1266	Human_Mind_and_Conditions-154	quarrel	v	10b1.81		
+Hale-1973-1798-1266	Human_Mind_and_Conditions-154	quarrel	v	10b1.81	1848	QUARREL
```

Comment:
This should be mapped to `3302 QUARREL (VERB)` https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L3288